### PR TITLE
Update SQL support docs for Blackhole

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Adding view and materialized view support. 

I am not sure we can claim this as correct since ALTER VIEW and ALTER MATERIALIZED view might not be supported.. if that is the case I have to list the supported commands separately.

## Additional context and related issues

Caused by missing docs in https://github.com/trinodb/trino/pull/18016

## Release notes

Release notes are already tackled with implementation PR.

### CODE REVIEW

**Review by vlad-lyutenko — APPROVED** [review]

**Comment by mosabua** [comment]
@vlad-lyutenko have you checked the code or tested that all view and MV statements are supported. Including ALTER ?

**Comment by vlad-lyutenko** [comment]
> @vlad-lyutenko have you checked the code or tested that all view and MV statements are supported. Including ALTER ?

No, we for my PR we needed only basic usages like get view and list all views

File: docs/src/main/sphinx/connector/blackhole.md [hunk 1]
```diff
@@ -124,9 +124,11 @@ additional features:
 - {doc}`/sql/drop-table`
 - {doc}`/sql/create-schema`
 - {doc}`/sql/drop-schema`
+- {ref}`sql-view-management`
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/blackhole.md:127]: `sql-view-management` contains `/sql/alter-view`. Blackhole connector doesn't support it.

Comment (mosabua) [reply docs/src/main/sphinx/connector/blackhole.md]: Thats what I thought from looking at the code.. thanks for confirming. Now I am just not sure about refresh materialized view ..

File: docs/src/main/sphinx/connector/blackhole.md [hunk 2]
```diff
@@ -14,11 +14,6 @@ In such cases, writes behave in the same way, but reads
 always return the specified number of some constant rows.
 You shouldn't rely on the content of such rows.
 
-:::{warning}
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/blackhole.md:17]: We dont ever support multiple coordinators anyway so this warning is redundant.

File: docs/src/main/sphinx/connector/blackhole.md [hunk 3]
```diff
@@ -124,9 +119,13 @@ additional features:
 - {doc}`/sql/drop-table`
 - {doc}`/sql/create-schema`
 - {doc}`/sql/drop-schema`
+- {doc}`/sql/create-view`
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/blackhole.md:122]: is show create view supported?

File: docs/src/main/sphinx/connector/blackhole.md [hunk 4]
```diff
@@ -124,9 +119,13 @@ additional features:
...
+- {doc}`/sql/drop-view`
+- {doc}`/sql/create-materialized-view`
+- {doc}`/sql/drop-materialized-view`
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/blackhole.md:125]: are refresh mv and show create mv supported?

**Comment by mosabua** [comment]
Updated and added some questions now @vlad-lyutenko and @ebyhr

File: docs/src/main/sphinx/connector/blackhole.md (hunk 3 above)
Comment (ebyhr) [reply docs/src/main/sphinx/connector/blackhole.md:122]: Yes.

**Review by ebyhr — APPROVED** [review]

File: docs/src/main/sphinx/connector/blackhole.md (hunk 3 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/blackhole.md:122]: Added

File: docs/src/main/sphinx/connector/blackhole.md (hunk 4 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/blackhole.md:127]: updated

**Comment by vlad-lyutenko** [comment]
https://github.com/trinodb/trino/pull/18628

**Pull request merged**